### PR TITLE
[receiver/podman] remove duplicate Timeout setting

### PR DESCRIPTION
**Pull request by codeboten**
Use the Timeout setting from ScraperHelper instead

### CODE REVIEW

File: receiver/podmanreceiver/factory.go [hunk 1]
```diff
@@ -34,7 +34,6 @@ func createDefaultConfig() *Config {
 	return &Config{
 		ScraperControllerSettings: cfg,
 		Endpoint:                  "unix:///run/podman/podman.sock",
-		Timeout:                   5 * time.Second,
```
Comment (atoulme) [inline receiver/podmanreceiver/factory.go:37]: Is 5s no longer default? If so need to update https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/podmanreceiver/README.md?plain=1#L35

Comment (TylerHelmuth) [reply receiver/podmanreceiver/factory.go:37]: The timeout is set to 5 seconds on like 32 (I missed it at first)

Comment (codeboten) [reply receiver/podmanreceiver/factory.go:37]: if you look at line 32 above, it's still the default

**Review by TylerHelmuth — APPROVED** [review]

**Comment by dmitryax** [comment]
podman receiver sets the timeout on the context which is already done by the scraper helper. So it can be removed I believe https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/eba9ae2e69df5235f7140aea3a86c4bbd5dfe9c4/receiver/podmanreceiver/podman.go#L175

File: receiver/podmanreceiver/factory.go (hunk 1 above)
Comment (atoulme) [reply receiver/podmanreceiver/factory.go:37]: ah that's silly. OK then.

**Review by atoulme — APPROVED** [review]

**Review by dmitryax — APPROVED** [review]

**Pull request merged**